quick-install/copy-certs.sh

```diff
@@ -3,16 +3,22 @@ set -eo pipefail
 unset CDPATH
 IFS=$'\n\t'
 
+namespace='kube-public'
+if [ ! -z "$1" ]
```

Comment:
doesn't necessarily need to be changed but "-n", in this case, would make a bit more sense

Reply:
True. I'll merge a PR if you open one :)